log/golog.go

```diff
@@ -0,0 +1,153 @@
+/*
+Copyright 2019 Caicloud Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package log
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+)
+
+// goLogger is the logger with go logger.
+type goLogger struct {
+	l         *log.Logger
+	level     Level
+	calldepth int
+}
+
+// NewGoStandardLogger creates a zap logger.
+func NewGoStandardLogger(level Level, out io.Writer, prefix string, flag int) Logger {
```

Comment:
Not sure if we should provide a NewDefaultGoStandardLogger since we already have one using stdLogger.

/cc @kdada

Reply:
Why do you wrap a standard log rather than output data to the `out` directly?

Reply:
The flag `stdlog.LstdFlags` is duplicated with the result of `prefix()`

Reply:
In my opinion, if there is a mature tool, prefer not to reinvent a new one. And std log supports log header line or some other features. WDYT

Reply:
> The flag `stdlog.LstdFlags` is duplicated with the result of `prefix()`

I think the prefix and stdFlags serves for different purposes. But it also works for me if we remove the support for prefix or flags.

Reply:
I think the behaviors of this logger should be splitted into two scenarios:
1. A standard implementation for std log.
2. A pure file logger for the log format of `prefix()`. 

So I think you can remove the `prefix()` from this PR and make it be the 1st implementation.

Reply:
Based on my personal preference, I'd like the 2nd implementation.

Reply:
SGTM

Reply:
BTW, can we output the log to file and to STDOUT? Maybe register two loggers.